hivemind/moe/client/beam_search.py

```diff
@@ -235,7 +237,11 @@ async def _get_active_successors(
         return successors
 
     def find_best_experts(
-        self, grid_scores: Sequence[Sequence[float]], beam_size: int, return_future: bool = False
+        self,
+        grid_scores: Sequence[Sequence[float]],
+        beam_size: int,
+        num_workers: Optional[int] = None,
```

Comment:
In which case do we want to call this function with a different num_workers than what was created? 

Moreover, this function (without an underscore) is only used in tests, so not sure if it's worth making it more complicated

Reply:
The reason i've added this is that num_workers was a __documented__ parameter in this function.

Reply:
alternatively, i could rm it from docstring

Reply:
[rm-ed]